CMakeLists.txt

```diff
@@ -1229,12 +1229,21 @@ else()
     target_link_libraries(${IMPL_TARGET} PRIVATE common ${LINKER_FLAGS} ${EXTRA_LIBS} ${MATH_LIB})
 
     if(APPLE AND ALSOFT_OSX_FRAMEWORK)
+        if(GIT_FOUND)
+            EXECUTE_PROCESS(COMMAND ${GIT_EXECUTABLE} rev-list --count head
+                TIMEOUT 5
+                OUTPUT_VARIABLE BUNDLE_VERSION
+                OUTPUT_STRIP_TRAILING_WHITESPACE
+                )
+        else()
+            set(BUNDLE_VERSION 1)
+        endif()
         set_target_properties(${IMPL_TARGET} PROPERTIES
             FRAMEWORK TRUE
             FRAMEWORK_VERSION C
             MACOSX_FRAMEWORK_IDENTIFIER "org.openal-soft.openal"
             MACOSX_FRAMEWORK_SHORT_VERSION_STRING ${LIB_VERSION}
-            MACOSX_FRAMEWORK_BUNDLE_VERSION 1
+            MACOSX_FRAMEWORK_BUNDLE_VERSION ${BUNDLE_VERSION}
```

Comment:
From what [I'm](https://cmake.org/cmake/help/v3.15/prop_tgt/MACOSX_FRAMEWORK_INFO_PLIST.html) [reading](https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundleversion), this should be a `major.minor.patch` version, such as from `LIB_VERSION`, rather than a commit count.

Reply:
Nope, from my mobile development experience, the CFBundleVersion should be a number, the means same with ```buildNumber```

Reply:
see the snapshot:
![image](https://user-images.githubusercontent.com/6977319/90758086-0c633980-e311-11ea-8b5a-44970f4ff08e.png)